lib/logux/stream.rb

```diff
@@ -19,8 +19,7 @@ def write(payload)
     private
 
     def process(payload)
-      return payload if payload.is_a?(::String)
-      payload.to_json
+      payload.is_a?(::String) ? payload : payload.to_json
```

Comment:
@wilddima why do we need to serialise string as not-json here?

Reply:
@dsalahutdinov sometime we should write invalid JSON, for example here:
https://github.com/wilddima/logux_rails/blob/fec57b289834145c9be0d825286d8ef23eb86d45/app/controllers/logux_controller.rb#L8

It's necessary for logux server.